stories/components/InputCollection.md

```diff
@@ -0,0 +1,40 @@
+# Carousel
+
+**Available components**:
+
+- [InputCollection](#inputcollection)
+
+**How it is built**
+
+- Use React's `useState` under the hood.
+
+## InputCollection
+
+Takes input type and can generate an array of inputs for that type (restricted to 'text' | 'email' | 'number' | 'url'
+The name provided is appended with index when outputting value (eg name[0]: "some value")
+
+### Props
+
+| name        | type   | default | description                |
+| ----------- | ------ | ------- | -------------------------- |
+| type        | string |         | Type of html input         |
+| placeholder | string |         | Placeholder for input      |
+| addVerbiage | string |         | Text for add button        |
+| onChange    | any    |         | optional onChange handler  |
+| name        | string |         | Name for key of the values |
+
```

Comment:
Can you add more details about `onChange`? Namely the fact that it is a callback that is called for all inputs individually and that the second parameter is the index of the individual input that is being changed.